node_modules/col-assets/lib/api.js

```diff
@@ -208,8 +209,14 @@ var incrementViewsIfRequired = function(asset, incrementViews, callback) {
       log.error({'err': err, 'id': id}, 'Failed to increase the views on an asset');
       return callback({'code': 500, 'msg': err.message});
     }
+    ActivitiesAPI.createActivity(ctx.course, ctx.user, 'view_asset', id, CollabosphereConstants.ACTIVITY.OBJECT_TYPES.ASSET, null, null, function(err) {
```

Comment:
Can you add a test that verifies that an asset view activity doesn't contribute to the engagement index score by default?